wgengine/monitor/polling.go

```diff
@@ -42,6 +43,15 @@ func (pm *pollingMon) Close() error {
 	return nil
 }
 
+func (pm *pollingMon) isCloudRun() bool {
```

Comment:
no need to be a method. The func doesn't access pm.

Reply:
Ended up adding a pm.logf().

Added more careful checks since Knative would match this as well, but Knative could actually have route changes so it would be necessary to poll regularly. Knative probably wouldn't get here because its RTMGRP support should work and it wouldn't need to fall back to polling, but it still seemed better to strengthen the check.